### ISSUE TITLE
Polling not working as outlined in the docs

### ISSUE THREAD

**Issue by fakingfantastic** [calebporzio/livewire#133]
**Describe the bug**
I'm following along with the demo under "Polling Actions" in the docs here: https://livewire-framework.com/docs/triggering-actions/, and I am not seeing the time refresh every 750ms. I'm kind of assuming it's just me, but maybe not? ¯\_(ツ)_/¯

**To Reproduce**
1. Make a component. I'm just reusing the Counter component because I'm lazy 

```php
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class Counter extends Component
{
    public function render()
    {
        return view('livewire.counter');
    }
}
```

2. Literally, like literally, lift the code from the docs for `counter.blade.php`
```php
<div wire:poll.750ms="$refresh">
    Current time: {{ now() }}
</div>
```
3. Get super bummed because it don't work 😢

**Expected behavior**
I should see that that sweet, sweet time update.

**Screenshots**
![Screen Recording 2019-08-06 at 07 50 AM](https://user-images.githubusercontent.com/20476/62537797-528d4e00-b81f-11e9-8e2c-fe44e95f11dd.gif)

**Desktop (please complete the following information):**
 - Chome 75 / Mac

**Additional context**
Tested against dev-master at the time of writing (5dd8cbc9)[https://github.com/calebporzio/livewire/commit/5dd8cbc94631a893866fbacbfc1777ec6c8140ae]

**Comment by davidpiesse** [comment]
I get the same behaviour... interesting....
Went and looked at it again and I had the code outside the root div.

**Comment by fakingfantastic** [comment]
Ahh... I figured it out.

Drilled down into `node_initializer.js` and saw that the `walk` skips the root node. If you follow along blindly from the docs (like 👍 this guy 👍), you will drop the code snippet in a component verbatim, but you miss the fact that in (Quick Start)[https://livewire-framework.com/docs/quickstart/] `@livewire('counter')` needs to be wrapped in a div, then it breaks a bunch downstream.

@calebporzio - Im gonna make a PR to add some text to make a bigger deal about that root node in the Quickstart

**Comment by calebporzio** [comment]
That would be great @fakingfantastic, thanks !

**Comment by fakingfantastic** [comment]
@calebporzio Added that one this morning and some other edits yesterday https://github.com/calebporzio/livewire-docs/pulls 🎅

**Comment by calebporzio** [comment]
That's great, thanks.

However, I think the root div is the best (and most obvious) place to register `wire:poll`.

I added it and it will be available in the next release.

**Comment by calebporzio** [comment]
This should be fixed in v0.1.6
